cloudnet/src/main/java/de/dytanic/cloudnet/console/ConsoleReadThread.java

```diff
@@ -0,0 +1,61 @@
+package de.dytanic.cloudnet.console;
+
+import de.dytanic.cloudnet.common.concurrent.CompletableTask;
+import de.dytanic.cloudnet.common.concurrent.ITask;
+import de.dytanic.cloudnet.console.animation.AbstractConsoleAnimation;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jline.reader.EndOfFileException;
+import org.jline.reader.UserInterruptException;
+
+import java.util.function.Consumer;
+
+public class ConsoleReadThread extends Thread {
+
+    public ConsoleReadThread(JLine3Console console) {
+        this.console = console;
+    }
+
+    private final JLine3Console console;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5923591)